tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc

```diff
@@ -83,7 +83,6 @@
 #include <EnergyPlus/ZoneTempPredictorCorrector.hh>
 
 using namespace EnergyPlus;
-using namespace ObjexxFCL;
```

Comment:
This was the offending line because ObjexxFCL has an `fmt` function and so does the new thirdparty/fmt library, causing an ambiguity.